### ISSUE TITLE
Populate data to staging database

### ISSUE THREAD

**Issue by roll** [opentrials/opentrials#87]
Like mentioned here - https://github.com/opentrials/opentrials/issues/31- we could have docker-cloud apps for different environments.

@vitorbaptista please send me db creds when will be on it

**Comment by vitorbaptista** [comment]
Agreed, we need a good subset of data for staging and even local development. I added some seeds in https://github.com/opentrials/api/blob/master/seeds/trials.js, but it's still too little.

**Comment by roll** [comment]
@vitorbaptista 
We don't write to `database` from API/UI - why do not use production database url for `api.next` and `explorer.next`?

**Comment by vitorbaptista** [comment]
@roll because of the database migrations

**Comment by roll** [comment]
oh right) So I suppose I need to do this first - https://github.com/opentrials/opentrials/issues/105 - where I'll be updating docker-cloud setup (and I'll be ably to add staging environment).

Let move it to backlog?

**Comment by pwalsh** [comment]
@vitorbaptista relevant?

**Comment by vitorbaptista** [comment]
@pwalsh Yes, we still need some good subset (or the complete dump) of production data to be able to try changes before merging into production.
